basekit/cpanel-plugin
# INT-430 Allow provide an icon when installing plugin

To test:
- Follow the instruction on the wiki to upload the code
- Check the icon on x3 and paper lantern is the BaseKit one
- Upload an icon to the cPanel server or use one already there (50px-Soccer_ball.svg.png , doniell15.png)
- open cpanel folder and run ./install.sh -icon [path to a 50x50 png icon] 
```
 ./install.sh -icon ../doniell15.png
```
- Check on x3 and paper lantern the new icon is there
- Run again ./install.sh (without parameter) and the original basekit icon should be there again.
